src/Elmish.WPF/Elmish.WPF.fsproj

```diff
@@ -15,12 +31,24 @@
     <Compile Include="Utils.fs" />
     <Compile Include="Cmd.fs" />
     <Compile Include="Program.fs" />
-    <None Include="paket.references" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Elmish" Version="[3.0.3, 4)" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-19367-01">
```

Comment:
What is `Microsoft.SourceLink.GitHub` for?  I could find any reference to it in `master`.  Did you mean to add this?

Reply:
Yes, that was intended. It is strictly speaking a different change, but I don't mind batching them together in this case.

It's for SourceLink support, meaning that when debugging client apps, it's possible to step into Elmish.WPF code.

Reply:
Re. SourceLink, see [dotnet/sourcelink](https://github.com/dotnet/sourcelink). I've never been able to get it working with symbol packages no matter what I try, so I just consistently embed PDBs in the main package.

Reply:
Oh, that sounds great!  It is on my list of "technical debt" for my current project at work to get that kind of debugging into our NuGet package dependencies working.